controllers/tas/statuses_test.go

```diff
@@ -51,7 +52,10 @@ func setupAndTestStatusNoComponent(instance *trustyaiopendatahubiov1alpha1.Trust
 var _ = Describe("Status and condition tests", func() {
 
 	BeforeEach(func() {
-		k8sClient = fake.NewClientBuilder().WithScheme(scheme.Scheme).Build()
+		k8sClient = fake.NewClientBuilder().WithScheme(scheme.Scheme).WithStatusSubresource(
+			&routev1.Route{},
+			&trustyaiopendatahubiov1alpha1.TrustyAIService{},
```

Comment:
Need to register the status subresoruce separately with the updated controller-runtime package.
Thanks @yhwang figured this out.